AWSCore/Service/AWSService.h

```diff
@@ -113,6 +113,9 @@ typedef NS_ENUM(NSInteger, AWSServiceErrorType) {
 
 - (void)addUserAgentProductToken:(NSString *)productToken;
 
+- (instancetype)initWithRegionName:(NSString *)regionName
```

Comment:
Do we need this method? I think it is better to keep it enum so that whoever use this method knows what all regions are supported, than giving a freeform string?

Reply:
Removed